cvat-core/src/ml-model.ts

```diff
@@ -157,27 +151,7 @@ Object.defineProperties(MLModel.prototype.getPreview, {
         writable: false,
         enumerable: false,
         value: async function implementation(): Promise<string | ArrayBuffer> {
-            if (this.provider === ModelProviders.CVAT) {
-                return '';
-            }
-            return new Promise((resolve, reject) => {
-                serverProxy.functions
-                    .getPreview(this.id)
-                    .then((result) => {
-                        if (isNode) {
-                            resolve(global.Buffer.from(result, 'binary').toString('base64'));
-                        } else if (isBrowser) {
-                            const reader = new FileReader();
-                            reader.onload = () => {
-                                resolve(reader.result);
-                            };
-                            reader.readAsDataURL(result);
-                        }
-                    })
-                    .catch((error) => {
-                        reject(error);
-                    });
-            });
+            return '';
```

Comment:
Do we need this method now?

Reply:
Yes, we still need to work with empty previews for Lambda functions

Reply:
Embedded functions don't have a preview. So, why we need this method?

Reply:
It looks strange that it just returns empty string.